src/components/VoucherModal/index.tsx

```diff
@@ -1,25 +1,42 @@
 import React, { useState, useEffect, useLayoutEffect } from "react"
 import { Skeleton, SkeletonText } from "@chakra-ui/react"
 
+import {
+  redeemVoucher
+} from "api/voucher"
+
 import usePrevious from "hooks/usePrevious"
 import { dateToString } from "utils/date"
 
-import { Modal, ModalProps, Heading, Text } from "@commitUI/index"
+import { Modal, ModalProps, Heading, Text, Button } from "@commitUI/index"
 
 import styles from "./VoucherModal.module.css"
 
 type Props = Omit<ModalProps, "children"> & {
   voucher?: Voucher
   isValidating: boolean
+  user?: User
+  redeemed?: String[] | undefined
+  onCloseHandler: VoidFunction
```

Comment:
Can you change the type to `() => void`. This is what we usually use for void functions